basin-helper

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PATH_TO_JSON=/tmp/basin.json
+PATH_TO_FILES=$1
+PATH_TO_SHARD=$2
```

Comment:
No, it can be anywhere, you can generate shards wherever you please and then test them with --content-path= option.